app/controllers/api/v1/webauthn_verifications_controller.rb

```diff
@@ -6,8 +6,12 @@ def create
       if user
         if user.webauthn_credentials.present?
           token = user.refresh_webauthn_verification.path_token
-
-          render yaml: { path: "example.com/webauthn/#{token}" }
+          webauthn_path = "example.com/webauthn/#{token}"
+          respond_to do |format|
+            format.yaml { render yaml: { path: webauthn_path } }
+            format.json { render json: { path: webauthn_path } }
+            format.any(:all) { render plain: webauthn_path }
```

Comment:
I found using Postman to hit this endpoint that for some reason if I don't specify an Accept header, this defaults to yaml, the first one on the list. So I think for this to work properly, format.any needs to be at the top.

Reply:
Interesting finding! It explains why the API key controller lays it out in that order. A little concerning that it wasn't detected by the functional test.

I'll reorder and then investigate how to get a failing test for this behaviour.

Reply:
A question that occurs to me: should we be writing an integration test case instead? Rails docs say functional test cases are discouraged, but there are far more functional test cases in rubygems.org than integration test cases.

Reply:
That's a good question. I think if writing an integration test for this use case would catch things that a functional test would not, then we should add it. But if we are just aiming to improve our test suite overall, we should save that for a separate PR

Reply:
Makes sense. I've been playing with the integration test today, but only at a very preliminary stage.